pkg/model/fileshare.go

```diff
@@ -0,0 +1,102 @@
+package model
+
+// FileShareSpec is a schema for fileshare API. Fileshare will be created on some backend
+// and can be shared among multiple users.
+
+type FileShareSpec struct {
+	*BaseModel
+
+	// The uuid of the project that the fileshare belongs to.
+	TenantId string `json:"tenantId,omitempty"`
+
+	// The uuid of the user that the fileshare belongs to.
+	UserId string `json:"userId,omitempty"`
+
+        // The uuid of the fileshare.
+        FileShareId string `json:"fileshareId,omitempty"`
```

Comment:
What is the relationship between FileShareId and Id in BaseModel?

Reply:
I think this field should be removed since BaseModel already defines Id.

Reply:
FileShareId removed from fileshare.go
Thanks